bundles/framework/announcements/Tile.js

```diff
@@ -0,0 +1,54 @@
+/*
+ * @class Oskari.mapframework.bundle.layerlist.Tile
+ *
+ * Renders the "announcements" tile.
+ */
+const BasicTile = Oskari.clazz.get('Oskari.userinterface.extension.DefaultTile');
```

Comment:
Is there a reason for implementing a custom tile instead of just using DefaultTile?